Airports/EGJJ/Ownership.txt

```diff
@@ -30,18 +30,59 @@ ALTOWNER:Mentoring TWR/APP:JJAM:JJA:JM:J:LW:LWN:LWC:LWS:L
 BORDER:JJAPP
 ARRAPT:EGJJ
 
-SECTOR:JJCTR:0:19500
+SECTOR:JJAPP-M:0:3000
```

Comment:
Should these two regions not replace the JJAPP:0:8500 above? If so, this will likely affect COPX